src-cpp/rdkafkacpp.h

```diff
@@ -1216,6 +1216,13 @@ class RD_EXPORT Conf {
                                          const void *buffer, size_t size,
                                          std::string &errstr) = 0;
 
+  /** @brief Set callback_data for ssl engine
+   *
+   * @returns CONF_OK on success, else CONF_INVALID.
+   */
+  virtual Conf::ConfResult set_engine_callback_data(void *value,
```

Comment:
style-nit: space before paren.

Add `@remark The memory pointed to by \p value must remain valid for the lifetime of the configuration object and any Kafka clients that use it.`